README.md

```diff
@@ -72,23 +72,35 @@ refer to the [online documentation](https://documentation.centreon.com/docs/cent
 
 First of all, check if you have these packages installed (Note that packages names come from Centos 7 distribution, so if some packages names don't match on your distribution try to find their equivalent names) : 
 	
-    git, make, cmake, gcc-c++, python3, python3-pip3, lua-devel, rrdtool-devel, gnutls-devel.
+    git, make, cmake3, gcc-c++, python3, python3-pip3, lua-devel, rrdtool-devel, gnutls-devel, mariadb-devel, perl-ExtUtils-Embed.
 
 If they are not installed, please install them.
 
+For the projet compilation you need to have conan installed. To install conan you need pip3 if you are using python3 (python package manager). You can install conan like that.
+
+    $> pip3 install conan
+
 If you are on a Centos 7 distribution, follow these steps :
    
     $> git clone https://github.com/centreon/centreon-collect
-    $> cd centreon-collect && ./cmake.sh
+    $> cd centreon-collect
+    $> mkdir build
+
+You will need to use cmake3, if you already had cmake just do those commands to use cmake3 instead of cmake by renaming it like so :
```

Comment:
Because of several packages provided by conan, you need the  version 3 of cmake executable to be named cmake. But on some distributions like centos7, the version 3 of cmake is called cmake3. For a such distribution, execute the following procedure: